docs/glossary.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Glossary
-description: zkApps (zero knowledge apps) are Mina Protocol smart contracts powered by zero knowledge proofs, specifically using zk-SNARKs. Use this quickstart guide to deploy zkApps and resources for learning TypeScript.
+description: A zero knowledge list of terms with a focus on the Mina ecosystem. 
```

Comment:
I think 'zero knowledge list' is fun but not so accurate and perhaps a bit of a stretch. What do you think of this suggestion?

```suggestion
description: A list of terms relating to zero knowledge and the Mina ecosystem. 
```

Reply:
way better, it's always better together, thank you!